merlin/core/compat.py

```diff
@@ -17,9 +17,16 @@
 try:
     from numba import cuda
 
+except ImportError:
+    cuda = None
```

Comment:
Might need to add the following line at the top of the file now. To avoid the linter raising errors about us not using cuda in here anymore

```
# pylint: disable=unused-import
```

Reply:
Thanks! (forgot I didn't have pre-commit installed)